### PR TITLE
PMM-4309 Adding PMM2-Upgrade Testing Job

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: pmm/pmm2-upgrade-tests.groovy [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                    sauceconnect(options: '', sauceConnectPath: '') {
+                        sh """
+                            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+                            . ~/.nvm/nvm.sh
+                            nvm install 10.6.0
+                            sudo rm -f /usr/bin/node
+                            sudo ln -s ~/.nvm/versions/node/v10.6.0/bin/node /usr/bin/node
+                            npm install
+                            node -v
+                            npm -v
```
Comment (hors) [inline pmm/pmm2-upgrade-tests.groovy]: do you really need it ? or it was added for debug ?

**Comment by AlekSi** [comment]
What's the status of this PR?

File: pmm/pmm2-upgrade-tests.groovy (hunk 1 above)
Comment (puneet0191) [reply pmm/pmm2-upgrade-tests.groovy]: we would need it everytime, some issue with NPM and node, hence I have added it permanently

**Comment by puneet0191** [comment]
I will merge it once Slava gets his clarification, I missed his review.

**Pull request closed without merging**